riemann_client/client.py

```diff
@@ -6,7 +6,11 @@
 
 from __future__ import absolute_import
 
+import logging
 import socket
+from threading import RLock
+from threading import Timer
```

Comment:
How about making that client availability conditional on threading importing successfully?